### DISCUSSION TITLE
Xfce - login twice after suspend

### DISCUSSION THREAD

**Discussion by nomad101** [SpiralLinux/SpiralLinux-project#490] [Q&A]
Hi, I'm using spiralinux with Xfce. 

After suspending the system, I've set it for security that I would need to login into the session again. However, after logging in, after a few seconds I'm asked to log in again. I've looked at the power manager settings but can't figure out what's going on. 

thanks

**Comment by marwenius** [comment]
Seems to be a common problem:
https://github.com/the-cavalry/light-locker/issues/118

A somewhat radical solution:
https://forums.linuxmint.com/viewtopic.php?t=321251

But maybe this will help:
https://www.reddit.com/r/xfce/comments/11pshwh/comment/jc10y5r/

**Reply by geckolinux** [reply]
Hi there,

> After suspending the system, I've set it for security that I would need to login into the session again

I'm not sure what you changed, because screen locking after resume from suspend is the default setting for SpiralLinux XFCE, and I've never seen a double login screen.
